### ISSUE TITLE
Get RVSW to compile on MacOS

### ISSUE THREAD

**Issue by nickchan2** [angry-goose-initiative/wiki#44]
We are yet to successfully compile RVSW on MacOS. Not sure if there's something wrong with my environment or if there's something in the project that needs to be changed. Would be nice to get this working since a few of us have Macs.

**Comment by JZJisawesome** [comment]
Agreed. Nick would you be willing to try to tackle this again? I'd help but I don't have a mac. Though perhaps I could use my work macbook to test (I'd have to ask) :)

**Comment by nickchan2** [comment]
Yeah, I'll see what I can do

**Comment by nickchan2** [comment]
Low priority right now, will likely get back to this when Conductor adds RVSW
